perf.groovy

```diff
@@ -4,151 +4,308 @@ import jobs.generation.*;
 
 def project = GithubProject
 def branch = GithubBranchName
-def projectFolder = Utilities.getFolderName(project) + '/' + Utilities.getFolderName(branch)
+def projectName = Utilities.getFolderName(project)
+def projectFolder = projectName + '/' + Utilities.getFolderName(branch)
 
 def static getOSGroup(def os) {
-    def osGroupMap = ['Ubuntu':'Linux',
+    def osGroupMap = ['Ubuntu14.04':'Linux',
         'RHEL7.2': 'Linux',
         'Ubuntu16.04': 'Linux',
         'Debian8.4':'Linux',
-        'Fedora23':'Linux',
+        'Fedora24':'Linux',
```

Comment:
I would probably just add Fedora24 and Ubuntu here instead of replacing it.  Seems safer then replacing something.